README.md

```diff
@@ -256,7 +256,8 @@ shown below. Can't see yours? Have a look at the [extra platform instructions](h
    #### Nushell
 
    **Warning** This will change in the future. Only nu version v0.33 or higher is supported.
-   Add the following to your nu config file:
+   Add the following to your nu config file. You can check the location of this
+   file by running `config path` in nushell.
```

Comment:
Nu seems to be called nushell in titles but nu in text, so I'd prefer going with nu like in the previous sentence.
```suggestion
   Add the following to your nu config file. You can check the location of this
   file by running `config path` in nu.
```